cypress/support/step_definitions/getterSetterCalls.md

```diff
@@ -1,132 +1,327 @@
-
-### Examples:
- * `And 'Firebolt' platform triggers '1st party app' 'CLOSEDCAPTION_SETTINGS' event`
- * `And 'Firebolt' platform triggers '1st party app' event`
- * `And 'Firebolt' platform triggers '3rd party app' 'CLOSEDCAPTION_SETTINGS' event`
- * `And 'Firebolt' platform does not trigger '3rd party app' 'CLOSEDCAPTION_SETTINGS' event`
- * `And 'Firebolt' platform triggers '1st party app' event`
- * `And 'Firebolt' platform triggers '1st party app' 'CLOSEDCAPTION_SETTINGS' event with 'INVALID_TYPE_PARAMS'`
-
-## Given '(.+)' platform responds to '([^']*)'(?: '([^']*)')? (get|set) API(?: with '(.+)')?
-### Purpose: Performing a validation against the source of truth for the given API response
-
-### Params:
-| Param | Definition |
-| --- | --- |
-| sdk | name of the sdk |
-| appId | app identifier |
-| fireboltCallKey | key name passed to look for firebolt call object in fireboltCallData Json |
-| methodType | Determines which method doing content validation Ex: set or get |
-| errorContent | Doing error content validation when error content object key passed. Ex: 'INVALID_TYPE_PARAMS' |
-
-### Examples:
- * `And 'Firebolt' platform responds to '1st party app' 'CLOSEDCAPTION_SETTINGS' get API`
- * `And 'Firebolt' platform responds to '1st party app' 'CLOSEDCAPTION_SETTINGS' set API`
- * `And 'Firebolt' platform responds to '3rd party app' 'CLOSEDCAPTION_SETTINGS' get API`
- * `And 'Firebolt' platform responds to '1st party app' set API`
- * `And 'Firebolt' platform responds to '1st party app' 'CLOSEDCAPTION_SETTINGS' set API with 'INVALID_TYPE_PARAMS'`
-
-## '(.+)' platform (triggers|does not trigger) '(.*?)'(?: '(.*?)')? event(?: with '(.+)')?
-
-### Purpose: Performing a event validation against the source of truth
-
-### Params:
-| Param | Definition |
-| --- | --- |
-| sdk | sdk name |
-| eventExpected | Determines whether the event is expected or not. |
-| appId |app identifier |
-| fireboltCallKey | key name passed to look for firebolt call object in fireboltCallData |
-| errorContent | Doing error content validation when error content object key passed. Ex: 'INVALID_TYPE_PARAMS' |
-
-### Examples:
- * `And 'Firebolt' platform triggers '1st party app' 'CLOSEDCAPTION_SETTINGS' event`
- * `And 'Firebolt' platform triggers '1st party app' event`
- * `And 'Firebolt' platform triggers '3rd party app' 'CLOSEDCAPTION_SETTINGS' event`
- * `And 'Firebolt' platform does not trigger '3rd party app' 'CLOSEDCAPTION_SETTINGS' event`
- * `And 'Firebolt' platform triggers '1st party app' event`
- * `And 'Firebolt' platform triggers '1st party app' 'CLOSEDCAPTION_SETTINGS' event with 'INVALID_TYPE_PARAMS'`
+## Background
+
+FCS supports the creation of Firebolt call objects, which include methods, parameters, and content. These objects provide details to glue codes for making API calls and validating responses. They are designed to handle multiple method calls for repetitive scenarios.
```

Comment:
It would be good to link to a document on the Firebolt Call objects here.

Reply:
The first two sentences sound like generic info about FireboltCall objects but the last one: `They are designed to handle multiple method calls for repetitive scenarios.` - sounds more targeted at using the runtime dynamic objects. Can that be made clear in that sentence.

Reply:
addressed the comment